common/JackAtomic.h

```diff
@@ -21,6 +21,15 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
 #define __JackAtomic__
 
 #include "JackTypes.h"
+
+#ifdef WIN32
+#include "atomic_msvc.h"
+#define COMPARE_EXCHANGE(ADDRESS, NEW, EXPECTED) atomic::msvc::interlocked<UInt32, 4>::compare_exchange(ADDRESS, NEW, EXPECTED)
+#else
+#define COMPARE_EXCHANGE(ADDRESS, NEW, EXPECTED) CAS(EXPECTED, NEW, ADDRESS)
+#endif
+
```

Comment:
Wouldn't it be cleaner if you made a special CAS macro for _MSV_VER in windows/JackAtomic_os.h instead of introducing another macro/inline function here?